Asterisk.2013/Asterisk.NET/Common.cs

```diff
@@ -26,7 +27,29 @@ public static class Common
         public static char INTERNAL_ACTION_ID_DELIMITER = '#';
 
         /// <summary> Variables delimiter </summary>
-        public static char[] VAR_DELIMITER = {'|'};
+        public static Dictionary<string, char[]> VAR_DELIMITERS = new Dictionary<string, char[]>();
+
+        /// <summary> Variables delimiter getter </summary>
+        public static char[] GET_VAR_DELIMITER(string hostname)
+        {
+            if (!VAR_DELIMITERS.ContainsKey(hostname))
+            {
+                VAR_DELIMITERS.Add(hostname, new char[] { '|' });

```

Comment:
Yes, we could just return a default value if the key doesn't exist.

I think we must return something strictly because it is static, so there is no guarantee that "determineVersion" has been run prior.  Ideally, this variable should probably be moved into the ManagerConnection class, as the value is really indeterminate until "determineVersion" has been run.